pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -59,18 +63,19 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     patchShebangs share/extensions
-    patchShebangs fix-roff-punct
+    patchShebangs share/templates
+    patchShebangs man/fix-roff-punct
 
     # Python is used at run-time to execute scripts, e.g., those from
     # the "Effects" menu.
     substituteInPlace src/extension/implementation/script.cpp \
-      --replace '"python-interpreter", "python"' '"python-interpreter", "${python3Env}/bin/python"'
+      --replace '"python3"' '"${python3Env}/bin/python"'
```

Comment:
This will only work on Mac. I would prefer using patch from my PR as substitutions do not fail fast.

Reply:
This also works for Linux, why do you think this works only for macOS?

Reply:
https://gitlab.com/inkscape/inkscape/-/blob/INKSCAPE_1_0_RC1/src/extension/implementation/script.cpp#L74-88

Reply:
Looks like they changed it up in https://gitlab.com/inkscape/inkscape/-/commit/7164e93e3c19bc206b73267eb3b0805c2e7597fa but that is not in the RC1.

Reply:
Thanks, I'll fix this.

Reply:
Addressed in  863becb